pkg/globalnet/controllers/types.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/submariner-io/submariner/pkg/globalnet/controllers/ipam"
 )
 
+const ClusterGlobalEgressIPName = "cluster-egress-ip.submariner.io"
```

Comment:
In the enhancement PR we have used `cluster-default`, but if you think `cluster-egress-ip` is a better name, we can do that. However, since it could be more than a single IP, what do you suggest?

Reply:
We can omit the "_ip_" part, "_cluster-egress.submariner.io_".  I qualified it with our domain to further ensure uniqueness as we do with label names.  I saw that u/s qualifies well-known CR names with "_k8s.io_".

Reply:
Updated in last push.